### DISCUSSION TITLE
Switch after switch

### DISCUSSION THREAD

**Discussion by Patrick-Smits** [fabianishere/udm-iptv#7] [Q&A] [answered]
Quick question: Will this solution also work when I have the dream machine connected to a switch (meterkast) which is connected to another switch (woonkamer)? I will plug the IP TV Box in the second switch and will assign a vlan to that port. For the first switch I will assign all profiles to all ports. Will that work so that the IP TV Box can be used on the second switch? I know that the dreammachine as 8 ports but that is not enough for me in the meterkast. I need a swtich in there as well.

**Comment by fabianishere** [comment, marked as answer]
Yes, that will work. I recommend enabling IGMP Snooping on the switches in between the IP TV box and the Dream Machine to reduce the amount of multicast traffic in your network.

**Reply by Patrick-Smits** [reply]
Thank you for the fast reply! I am about to get fiber of KPN so will definitely use this tutorial!
